microsoft/PowerToys
# [FancyZones] Update windows positions after changing the layout fix

## Summary of the Pull Request

**What is this about:**

Fix window size and position update after changing a layout. 

**What is included in the PR:** 

**How does someone test / validate:** 

* Make sure the `During zone layout changes, windows assigned to a zone will match new size/position` option is on
* Snap window to a zone
* Change layout
* Verify window position and size were updated 

## Quality Checklist

- [x] **Linked issue:** #18679

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
